examples/satellite-azure/cluster.tf

```diff
@@ -0,0 +1,49 @@
+#####################################################
+# IBM Cloud Satellite -  Azure Example
+# Copyright 2021 IBM
+#####################################################
+
+###################################################################
+# Create satellite ROKS cluster
+###################################################################
+module "satellite-cluster" {
+  //Uncomment following line to point the source to registry level module
```

Comment:
unnecessary comment, should probably be moved to above 13 and fixed wording wise

Reply:
Comment removed